cleanlab_studio/utils/tlm_lite.py

```diff
@@ -175,8 +189,6 @@ def _score(
 
         if isinstance(score_response, dict):
             return {"response": response, **score_response}
-        elif isinstance(score_response, (float, int)):
-            return {"response": response, "trustworthiness_score": score_response}
         else:
             raise ValueError(f"score_response has invalid type {type(score_response)}")
```

Comment:
If `score_response` is always expected to be a `dict` then we don't need the else statement anymore

Reply:
this is also just trying to pass the mypy check, but i don't think it hurts to add the check and raise